src/main/java/Sample.java

```diff
@@ -273,7 +273,7 @@ public Sample(String brokerUrl, String clientId, boolean cleanSession, boolean q
     } catch (MqttException e) {
       e.printStackTrace();
       log("Unable to set up client: " + e.toString());
-      System.exit(1);
+      return;

```

Comment:
maybe this file should be deleted entirely?

Reply:
agreed ... dropped there chasing squirrels most likely ... deleted